### ISSUE TITLE
[Feature Request]: A list showing all GUID-specific SAI entries for the currently selected entry

### ISSUE THREAD

**Issue by Gultask** [azerothcore/Keira3#2158]
At the moment to search to see if a creature has a GUID-Specific SAI. I need to open a SQL Client and search each guid.
A list would provide quick-access, maybe it could also link to the editor itself

**Comment by Helias** [comment]
I do not think that is an immediate feature to do, but would be great, thanks for the suggestion

**Comment by Helias** [comment]
we could add a button like "SAI" next-to the Spawns list like this

<img width="217" alt="image" src="https://user-images.githubusercontent.com/519778/219941926-afbcef12-628c-42b8-b7dd-92452d6b3cc7.png">


or, we could even put a list of GUID -Specific SAI under the SAI editor when you edit the entry of a creature
(under the creature's timed actionlist)

![image](https://user-images.githubusercontent.com/519778/219942025-ba84eb64-fe4e-4ba3-9fba-3faa71582160.png)


What do you think? this is probably not hard to do

**Comment by Gultask** [comment]
I think both would be good 😛
The one in the spawns tab would allow players to quickly see which ones have it or not, without swapping between the Spawns tab and the SmartAI tab. But the SmartAI tab could also act as a link to the editor and quickly show what they do and the purpose of the SAI.
